### PR TITLE
Add metrics adapter gem to filestore

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello Card](https://trello.com/c/5cXWL11V/342-request-delay-between-pages)

## Description

This PR adds the [metrics adapter gem](https://github.com/ministryofjustice/metrics-adapter) - the [first version](https://github.com/ministryofjustice/metrics-adapter/pull/1).

## Caveats

I am considering 1 second as a slow request and I added via config of the gem.

## Next PR

- [ ] Add the mixpanel secret to our git crypt repo.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -235,7 +248,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.6.4p104
+   ruby 2.6.5p114
```
Comment (brenetic) [inline Gemfile.lock:251]: does this need to also be set in the Dockerfile?

File: config/initializers/general_metrics.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+require 'metrics_adapter/rails'
```
Comment (brenetic) [inline config/initializers/general_metrics.rb:1]: can we change the file name to `metrics_adapter`?

Comment (tomas-stefano) [reply config/initializers/general_metrics.rb]: Done!

File: Gemfile.lock (hunk 1 above)
Comment (tomas-stefano) [reply Gemfile.lock:251]: Yes it does. Although maybe it is done in a separate PR? I added a card on the triage https://trello.com/c/HcXuQxna/362-remove-ruby-version-definition-from-gemfilelock

**Pull request merged**